utils/utils.go

```diff
@@ -61,3 +62,15 @@ func MergeStringMaps(map1 map[string]string, map2 map[string]string) map[string]
 	}
 	return combined
 }
+
+// PrettyPrintOrExit writes a JSON serialized version of the content to stdout.
+// If a failure occurs during marshaling, the error is logged along with a
+// formatted version of the object and the program exits with a failure status.
+func PrettyPrintOrExit(content interface{}) {
```

Comment:
Perhaps make whether or not to exit an argument, that way this method will still work for the other use-cases throughout this PR which need this behavior but with a returned error.

Reply:
Or else, have a separate method which returns an error, and have both use a private helper.

Reply:
The second way sounds cleaner the more I think about it.

Reply:
I like that, it'll also go cleanly with the way the rest of this software is structured.